unstable/wlr-export-dmabuf-unstable-v1.xml

```diff
@@ -0,0 +1,230 @@
+        If the frame is no longer valid after the "frame" event has been called,
+        this callback will be used to inform the client to scrap the frame.
+        Source is still valid for as long as the subscription function does not
+        return NULL.
+        This may get called if for instance the surface is in the process of
+        resizing.
+      </description>
+    </event>
+
+    <request name="destroy" type="destructor">
+      <description summary="delete this object, used or not">
+        Unreferences the frame, allowing it to be reused. Must be called as soon
+        as its no longer used.
+      </description>
+    </request>
+  </interface>
+
+  <interface name="zwlr_export_dmabuf_manager_v1" version="1">
+    <description summary="manager to inform clients and begin capturing">
+      This object is a manager which informs clients about capturable windows
+      and is able to create callbacks from which to begin to receive content
+      from. The "title" argument in the "surface_info" event shall be used
+      to provide a user-readable identifier such as a window title or
+      program name.
+    </description>
+
+    <event name="surface_info">
+      <description summary="get valid surfaces for capture">
+        This will be called whenever a surface that's able to be captured
+        appears.
+      </description>
+      <arg name="title" type="string"
+           summary="window title or an identifier"/>
+      <arg name="id" type="uint"
+           summary="an identifier to create a context with"/>
+    </event>
+    <event name="surface_unavailable">
+      <description summary="notifies of a surface being unavailable to capture">
+        Called if a surface becomes unavailable to capture, for example if has
+        been closed.
+      </description>
+      <arg name="id" type="uint"
+           summary="an identifier of the surface"/>
+    </event>
+
+    <request name="capture_client">
+      <description summary="subscribe to start capturing">
+        Request to start capturing from a surface with a given id.
+        If an ID becomes unavailable, a NULL will be returned.
+      </description>
+      <arg name="frame" type="new_id"
+           interface="zwlr_export_dmabuf_frame_v1"/>
+      <arg name="id" type="uint"
+           summary="an identifier to start capturing from"/>
+      <arg name="overlay_cursor" type="int"
+           summary="overlay custom client hardware cursor on top of the frame"/>
+    </request>
+    <request name="capture_output">
+      <description summary="subscribe to start capturing">
+        Request to start capturing from an entire wl_output.
+        If an output becomes unavailable, a NULL will be returned.
+      </description>
+      <arg name="frame" type="new_id"
+           interface="zwlr_export_dmabuf_frame_v1"/>
+      <arg name="out" type="object" interface="wl_output" allow-null="false"/>
```

Comment:
`out` is a little vague, generally arguments that take an output are name `output`. `allow-null` is already set to `false` by default.